burnman/output_seismo.py

```diff
@@ -186,9 +186,9 @@ def write_axisem_input(layers, modelname='burnmanmodel_foraxisem', axisem_ref =
 
     if plotting:
         
-        plt.plot(table[:,0] / 1.e3, table[:,2] / 1.e3, color='g', linestyle='-')
-        plt.plot(table[:,0] / 1.e3, table[:,3] / 1.e3, color='b', linestyle='-')
-        plt.plot(table[:,0] / 1.e3, table[:,1] / 1.e3, color='r', linestyle='-')
+        plt.plot(table[:,0] / 1.e3, table[:,2] / 1.e3, color='g', linestyle='-', label = 'V_p')
+        plt.plot(table[:,0] / 1.e3, table[:,3] / 1.e3, color='b', linestyle='-', label = 'V_s')
+        plt.plot(table[:,0] / 1.e3, table[:,1] / 1.e3, color='r', linestyle='-', label = 'density')
```

Comment:
Can you change all these to have no spaces either side of the equals sign? That will keep flake8 happier (https://flake8.pycqa.org/en/latest/).